src/main/resources/io/jenkins/plugins/customizable_header/SystemMessage/config.jelly

```diff
@@ -3,13 +3,27 @@
   <f:entry>
     <f:textarea field="message"/>
   </f:entry>
-  <f:entry field="level" title="${%Background of System Message}">
-    <f:enum>${it.toString()}</f:enum>
+  <f:entry title="${%Background of System Message}">
+    <!-- <f:enum>${it.getText()}</f:enum> -->
+    <div class="jenkins-radio">
+      <input type="radio" name="level" value="info" class="jenkins-radio__input"
```

Comment:
should be doable with a foreach over the enum I would expect to have much simpler code

Reply:
done